### PR TITLE
replace hardcoded URLs and other strings

### PR DESCRIPTION
**Pull request by crissoca**
Fix for #69 Replace hardcoded origin URLs

### CODE REVIEW

**Review by grgur — APPROVED** [review]

**Comment by jaygarcia** [comment]
@crissoca , when you have cycles, please look at the current merge conflict and resolve? :)

File: src/templates/ProjectLandingPage/index.js [hunk 1]
```diff
@@ -85,6 +85,12 @@ const DetailsPage = ({ pageContext: { project } }) => {
 
 DetailsPage.propTypes = {
   pageContext: PropTypes.shape({
+    site: PropTypes.shape({
+      siteMetadata: PropTypes.shape({
+        title: PropTypes.string,
+        url: PropTypes.string,
+      }),
+    }),
```
[the remark this answers is not in the document]
Comment (crissoca) [reply src/templates/ProjectLandingPage/index.js:93]: Fixed on 9d3c2f3c1ca779f5c7267949f767c3f36f6a85f2

**Review by rodriguezmanu — APPROVED** [review]

**Comment by rodriguezmanu** [comment]
@crissoca could you please update this one in order to merge?

File: src/templates/ProjectLandingPage/index.js [hunk 2]
```diff
@@ -59,13 +59,13 @@ const DetailsPage = ({ pageContext: { project } }) => {
         {project.route ? (
           <meta
             property="og:url"
-            content={`https://labs.moduscreate.com/${project.route}/`}
+            content={`${site.siteMetadata.url + project.route}/`}
```
Comment (crissoca) [inline src/templates/ProjectLandingPage/index.js:62]:
```suggestion
            content={`${site.siteMetadata.url}${project.route}/`}
```

File: src/templates/ProjectLandingPage/index.js [hunk 3]
```diff
@@ -59,13 +59,13 @@ const DetailsPage = ({ pageContext: { project } }) => {
...
           />
         ) : null}
         {project.image ? <meta property="og:image" content={getSrc(project.image)} /> : null}
 
         {project.route ? (
-          <link rel="canonical" href={`https://labs.moduscreate.com/${project.route}/`} />
+          <link rel="canonical" href={`${site.siteMetadata.url + project.route}/`} />
```
Comment (crissoca) [inline src/templates/ProjectLandingPage/index.js:68]:
```suggestion
          <link rel="canonical" href={`${site.siteMetadata.url}${project.route}/`} />
```

**Review by crissoca — COMMENTED** [review]
Quick updates

**Comment by rodriguezmanu** [comment]
@crissoca could you confirm `npm run lint` run without any issues on these changes?

**Comment by crissoca** [comment]
> @crissoca could you confirm `npm run lint` run without any issues on these changes?

@rodriguezmanu yeah actually that's why there are some new changes here that are out of scope, all good.

**Review by rodriguezmanu — APPROVED** [review]

**Pull request merged**